pyart/graph/radarmapdisplay_cartopy.py

```diff
@@ -19,6 +19,12 @@
 import matplotlib.pyplot as plt
 try:
     import cartopy
+
+    # Temporary fix for Cartopy issue #1120.
+    from matplotlib.axes import Axes
+    from cartopy.mpl.geoaxes import GeoAxes
+    GeoAxes._pcolormesh_patched = Axes.pcolormesh
```

Comment:
I would wrap this with a `hasattr(GeoAxes, ‘_pcolormesh_patched’)`

Reply:
Good point, I will make the change.